torch/csrc/autograd/python_variable.cpp

```diff
@@ -633,6 +630,84 @@ static PyObject* THPVariable_make_wrapper_subclass(PyObject*, PyObject* args, Py
   END_HANDLE_TH_ERRORS
 }
 
+
+static PyObject* THPVariable_make_sym_wrapper_subclass(PyObject*, PyObject* args, PyObject* kwargs) {
+  HANDLE_TH_ERRORS
+  // NB: pin_memory doesn't actually do anything
+  // TODO: strides variant?
+  static PythonArgParser parser({
+    "_make_wrapper_subclass(PyObject* cls, SymIntArrayRef size, SymIntArrayRef strides, int64_t? storage_offset=None, *, MemoryFormat? memory_format=None, ScalarType dtype=None, Layout layout=torch.strided, Device device=None, bool pin_memory=False, bool requires_grad=False)",
```

Comment:
How about keeping only one `_make_wrapper_subclass` and just having a second overload for PythonArgParser? Having it as an overload should also help reduce duplication in this variant of the function.

Reply:
It being long is a good reason not to copy paste right ;)